src/main/java/com/singh/astha/notification/service/exceptions/handler/GlobalExceptionHandler.java

```diff
@@ -29,8 +29,9 @@ public ResponseEntity<ResponseWrapper<Object>> handle(
             ResponseException exception
     ) {
         // FIXME: Update the Response Exception
-        ErrorResponse errorResponse = ErrorResponse.builder().build();
-        errorResponse.setMessage(exception.getMessage());
+        ErrorResponse errorResponse = ErrorResponse.builder()
+                .message(exception.getMessage())
+                .build();
```

Comment:
Use the factory method from() inside the ErrorResponse to create the instance.

Reply:
fixed